dependencies.gradle

```diff
@@ -74,6 +73,7 @@ ext {
             kotlin                : "org.jetbrains.kotlin:kotlin-stdlib-jdk8:${versions.kotlin}",
             kotlinReflect         : "org.jetbrains.kotlin:kotlin-reflect:${versions.kotlin}",
             lifecycleExtensions   : "androidx.lifecycle:lifecycle-extensions:${versions.lifecycle}",
+            lifecycleCommonJava8  : "androidx.lifecycle:lifecycle-common-java8:${versions.lifecycle}",
```

Comment:
Nice catch!

All `versions`, `libraries`, `testLibraries`, and `androidTestLibraries` were lexicographically sorted. Also Java 8 should be the default compiler across all modules.

Maybe `lifecycleCommon` would be more semantic/consistent name. Does it make sense @marukami?

Reply:
> were lexicographically sorted

Ah, I see. `lifecycleCommon` seems fine